release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -0,0 +1,46 @@
+[id='con-relnotes-fixed-issues_{context}']
+= Fixed issues
+
+This section lists fixed issues with {product} {product-version}:
+
+Fix an inability to dynamically load a number of upstream backend plugins::
++
+--
+Before this release, a whole category of upstream or third-party backstage backend plugins could not be loaded dynamically when converted to dynamic plugins. Impacted plugins were those using the `resolvePackagePath()` backstage API method. This issue impacted most of the backend plugins using the database service and providing migration scripts.
+This release removes such an important limitation of the dynamic plugin feature.
+--
+
+Incorrect configuration in the Atlassian Auth provider::
++
+--
+Before this release, the authorization URL was constructed as `${baseUrl}/oauth/authorize`, which did not align with the Atlassian expected path, resulting in an authentication error. This release fixes the issue. Now, the users are directed to the authorization URL to get an authorization code for successful authentication.
+--
+
+RBAC: CSV repopulates database whenever there is a server restart::
++
+--
+Before this release, the `policy.csv file` was used to load permissions into {product-short}. There might be instances of overlapping permissions, and the permissions could be reloaded into the database. This release fixes the issue.
```

Comment:
No need to put `This release fixes the issue.` for each fixed item as it's already under the fixed list section.